tensorflow/compiler/mlir/hlo/include/mlir-hlo-c/Attributes.h

```diff
@@ -275,6 +275,22 @@ MLIR_CAPI_EXPORTED bool mlirMhloAttributeIsAFusionKindAttr(MlirAttribute attr);
 MLIR_CAPI_EXPORTED std::string mlirMhloFusionKindAttrGetFusionKind(
     MlirAttribute attr);
 
+//
+// RngAlgorithmAttr.
+//
+// Creates a new RngAlgorithm attribute with the given 'algorithm' string
+// parameter.
+MLIR_CAPI_EXPORTED MlirAttribute
+mlirMhloRngAlgorithmAttrGet(MlirContext ctx, const std::string &kind);
+
+// Returns true if the given attribute is a RngAlgorithm attribute.
+MLIR_CAPI_EXPORTED bool
+mlirMhloAttributeIsARngAlgorithmAttr(MlirAttribute attr);
+
+// Returns the rng-algorithm string associated with RngAlgorithm attribute.
+MLIR_CAPI_EXPORTED std::string
```

Comment:
We shouldn't be using std::string in the C API, but there is existing behavior like this in this file which is already tracked to be removed and this wouldn't add much cost.

Reply:
I will change to const char*.